stdlib/types.pyi

```diff
@@ -563,6 +566,9 @@ def prepare_class(
     name: str, bases: tuple[type, ...] = (), kwds: dict[str, Any] | None = None
 ) -> tuple[type, dict[str, Any], dict[str, Any]]: ...
 
+if sys.version_info >= (3, 12):
+    def get_original_bases(__cls: Any) -> tuple[Any, ...]: ...
```

Comment:
It needs to be passed a type, right?

```suggestion
    def get_original_bases(__cls: type) -> tuple[Any, ...]: ...
```

Reply:
I'm really not having a good night lol, yeah you're right

Reply:
Actually should that be type[Any] or does it not matter?

Reply:
It shouldn't matter

Reply:
Oh yeah prepare_class doesn't subscript further